flows/contact.go

```diff
@@ -369,6 +370,11 @@ func (c *Contact) ReevaluateDynamicGroups(env envs.Environment, allGroups *Group
 func (c *Contact) QueryProperty(env envs.Environment, key string, propType contactql.PropertyType) []interface{} {
 	if propType == contactql.PropertyTypeAttribute {
 		switch key {
+		case contactql.AttributeID:
```

Comment:
this was missing.. I guess cuz we previously only used this code for dynamic groups which don't support id

Reply:
is that the problems?

Reply:
the problems?

Reply:
referring to the PR name, but see the other one too.